library/psa_crypto_core_common.h

```diff
@@ -0,0 +1,64 @@
+/**
+ * \file psa_crypto_core_common.h
+ *
+ * \brief Utility macros for internal use in the PSA cryptography core.
+ */
+/*
+ *  Copyright The Mbed TLS Contributors
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *  not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#ifndef PSA_CRYPTO_CORE_COMMON_H
+#define PSA_CRYPTO_CORE_COMMON_H
+
+/** Return an offset into a buffer.
```

Comment:
```suggestion
/** Return a buffer with addition offset.
```
Not sure whether the wrokds are right. But seems it should return a buffer.

Reply:
Amusingly this is just cloned from the original mbedtls version, but I kinda agree it doesn't sound right.

Perhaps "Return a pointer to an offset into a buffer"?

This probably goes for the constant version as well.

Reply:
As Paul mentioned this is just a clone of the Mbed TLS version. I prefer to keep them in sync and the way it is in this PR.